data_pipeline/elife_article_xml/elife_article_xml_pipeline.py

```diff
@@ -116,7 +120,10 @@ def get_article_json_data_from_xml_string_content(
                         if key not in ('related_article', 'article_id'):
                             article_meta_dict.pop(key, None)
                     LOGGER.info(article_meta_dict)
-                return article_meta_dict
+                return {
+                    'article_type': parsed_dict['article']['article_type'],
+                    **article_meta_dict
```

Comment:
I would usually put the expansion of the original dict first, that would make it clear that `article_type` is always set from this place. Otherwise if `article_meta_dict` contained an `article_type` key, it would be used instead. (It is probably impossible for this pipeline, but may not be obvious just reading the code)